qa/L0_java_memory_growth/Simple.java

```diff
@@ -394,6 +394,13 @@ output0, output1, new FloatPointer(input0_data), new FloatPointer(input1_data),
       } else {
         return false;
       }
+      if((memory_snapshots.get(index_90) / 1E6) >= 30){
+        System.out.println("Exceeded allowed memory (30MB), got " +
```

Comment:
The 30 MB should come from the constant variable. So that in future we can just change the constant value and all these messages would not need change.

Reply:
Fixed.